src/migrations/versions/cbbba858deb0_init.py

```diff
@@ -5,84 +5,110 @@
 """
 from alembic import op
 import sqlalchemy as sa
+from sqlalchemy import inspect
+
 import sqlmodel
 
+from config import app_config
 
 revision = "cbbba858deb0"
 down_revision = None
 branch_labels = None
 depends_on = None
 
+DATABASE_URL = app_config.database.url
+engine = sa.create_engine(DATABASE_URL)
+
 
 def upgrade() -> None:
-    op.create_table(
```

Comment:
Why this change? We should not change older migrations in any case. (Please also don't reformat them, to keep unnecessary git changes to minimum)

Reply:
Ok. I understand. I did this because the alembic upgrade head command in an existing environment causes errors that the table exists. So I check beforehand if the table exists.